retis/Cargo.toml

```diff
@@ -42,14 +42,14 @@ libbpf-sys = "1.5"
 libc = "0.2"
 log = { version = "0.4", features = ["std"] }
 memoffset = "0.9"
-nix = { version = "0.28", features = ["feature", "mount", "time", "user"] }
+nix = { version = "0.29", features = ["feature", "mount", "time", "user"] }
 once_cell = "1.15"
 pager = "0.16"
 pcap = "1.3"
 pcap-file = "2.0"
 plain = "0.2"
-pnet_packet = "0.34"
-rbpf = {version = "0.2", optional = true}
+pnet_packet = "0.35"
```

Comment:
We don't actually need to update the version but that's something we do regularly for a number of reasons:

- Upgrading crates has proven to be challenging in various examples and staying close to the latest one helps with not having a large gap.
- It helps discovering issues in the crates we use and helps when reporting the issue upstream early on.
- It helps with getting latest improvements and fixes.
- This is a common behavior for Rust packages IMO.

I understand from a packaging point of view this can introduce some hiccups but we're wearing the upstream hat  here.

(Also not upgrading can trigger similar incompatibility issues when packaging and downgrading a package in a distro is harder than upgrading to a newer version).

Hope that explains it.

Unrelated: we have a similar issue, but which is IMO also very different. We have a case where we do not want to upgrade a package because then Retis wouldn't compile on RHEL8. That's a compiler issue not a packaging one, so we can make the call upstream. But I'm taking the opportunity to raise this as it's a choice and should probably be discussed.

Reply:
Although the policy does make sense, its implications are that we could end up with big (potentially risky) refactors just before releasing, plus an extra delay for packaging if other dependencies need to be updated. 

This is a completely unrelated discussion but: Maybe we could schedule a dependency update in the middle of the release "cycle"?

Reply:
Yes, I think that makes perfect sense. We should try to upgrade more in between releases (maybe we could set up dependsabot for this) but less right before a release to stabilize things.

I had a look at the `pnet` latest release notes and it looks like there's nothing major for us in there. Let's keep using 0.24 then and do the bump right after the release.